src/index.js

```diff
@@ -1,2 +1,63 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function render(count = 0) {
+  function handleClick(value) {
+    const newValue = value + 1;
+
+    render(newValue);
+  }
+
+  function handleClickNumber(value) {
+    const newValue = value;
+
+    render(newValue);
+  }
+
+  const element = (
+    <div id="hello" className="greetoing">
+      <p>
+        <button type="button" onClick={() => handleClick(count)}>
```

Comment:
```suggestion
        <button type="button" onClick={handleClick}>
```

함수 내부 로직에서 count + 1 을 해주면 이렇게 작성이 가능할거에요 ;)